src/index.ts

```diff
@@ -38,8 +43,22 @@ const getRev = (osVersion: string) => {
 	}
 };
 
-const isDevelopmentVersion = (version: string) => {
-	return /(\.|\+|-)dev/.test(version);
+const isDevelopmentVersion = (parsedVersion: semver.SemVer | null) => {
+	if (parsedVersion === null) {
+		return false;
+	}
+
+	return parsedVersion.build.indexOf('dev') >= 0;
```

Comment:
I know that we tend to avoid this in favor of `_.includes`, but lodash isn't imported in this package and I was hesitant about adding it just for this case.